oxidation/libparsec/crates/crypto/src/sodiumoxide/secret.rs

```diff
@@ -91,16 +91,27 @@ impl SecretKey {
 
     pub fn from_password(password: &str, salt: &[u8]) -> Self {
         let mut key = [0; KEYBYTES];
-        let salt = Salt::from_slice(salt).expect("Invalid salt");
-
-        derive_key(
-            &mut key,
-            password.as_bytes(),
-            &salt,
-            OPSLIMIT_INTERACTIVE,
-            MEMLIMIT_INTERACTIVE,
-        )
-        .expect("Can't fail");
+
+        if cfg!(feature = "test-unsecure-but-fast-secretkey-from-password") {
```

Comment:
same as above

Reply:
```suggestion
        // During test we want to skip the `argon2` algorithm for hashing the password
        // Because it take sometime.
        // For that we replace argon with a very basic algorithm that copy the `password + salt` to the first key bytes.
        if cfg!(feature = "test-unsecure-but-fast-secretkey-from-password") {
```